assets/css/error-rescue.css

```diff
@@ -1,20 +1,25 @@
 .kite-error-rescue-sidebar {
-  --line-del-background: rgb(120, 40, 40);
```

Comment:
Maybe we should keep these variables in the corresponding theme files. There's on for each variant.